src/Results.svelte

```diff
@@ -105,7 +100,7 @@
     <div class="info-text">
       {#if selectedTab === STEUERSTUNDUNG}
         <p>
-          Ihr Bundesland <b>{$bundeslaender.find(land => land.id == selection.state).name || ''}</b> bietet
+          Ihr Bundesland <b>{selection.state ? $bundeslaender.find(land => land.id == selection.state).name || '' : ''}</b> bietet
```

Comment:
If selection.state is not set, the html code would look like this `Bundesland <b></b> bietet` so two white spaces before and after the `b` tags